packages/react-avatar/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "@fluentui/react-badge": "^9.0.0-alpha.64",
+    "@fluentui/react-icons": "^1.1.136",
```

Comment:
can we add this to root package json as devDep as well ? so anyone can consume icons within stories ?

- with that you can also remove this dep from vr-tests

Reply:
@Hotell it doesn't appear to have worked to remove the dependency from vr-tests. I'm going to add it back.